docs/tutorials/tutorial2.md

```diff
@@ -14,148 +14,49 @@ As reported in our paper, we trained an AtacWorks model to learn a mapping from
 
-```
-python $atacworks/scripts/main.py infer \
-    --files NK.50_cells.h5 \
-    --sizes_file $atacworks/data/reference/hg19.auto.sizes \
-    --config configs/infer_config.yaml \
-    --config_mparams configs/model_structure.yaml \
-```
-Note: `infer_config.yaml` is set up to use multiple GPUs. If you are using a single GPU, edit `infer_config.yaml` to change the line `gpu: "None"` to read `gpu: 0`. 
+See below for instructions to use our pre-trained models or your own trained models.
 
-The inference results will be saved in the folder `output_latest`. This folder will contain four files: 
-1. `NK_inferred.track.bedGraph` 
-2. `NK_inferred.track.bw` 
-3. `NK_inferred.peaks.bedGraph`. 
-4. `NK_inferred.peaks.bw`
+## Input files
 
-`NK_inferred.track.bedGraph` and `NK_inferred.track.bw` contain the denoised ATAC-seq track. `NK_inferred.peaks.bedGraph` and `NK_inferred.peaks.bw` contain the positions in the genome that are designated as peaks (the model predicts that the probability of these positions being part of a peak is at least 0.5)
+To denoise and call peaks from low-coverage/low-quality ATAC-seq data, you need three input files:
 
-To change any of the parameters for inference with the deep learning model, you can edit the parameters in `configs/infer_config.yaml` or `configs/model_structure.yaml` and run the command above. 
+1. A trained AtacWorks model file with extension `.pth.tar`.
 
-Type `python $atacworks/scripts/main.py infer --help` for an explanation of the parameters.
+2. A coverage track representing the number of sequencing reads mapped to each position on the genome in the low-coverage or low-quality dataset. This may be smoothed or processed in the same way as the files used for training the model. Format: [bigWig](https://genome.ucsc.edu/goldenPath/help/bigWig.html)
 
-If you are using your own model instead of the one provided, edit `configs/infer_config.yaml` to supply the path to your model under `weights_path`, in place of `model.pth.tar`.
-
-## Step 8: Format peak calls
-
-Delete peaks that are shorter than 20 bp in leangth, and format peak calls in BED format with coverage statistics and summit calls:
+3. Chromosome sizes file - a tab-separated text file containing the names and sizes of chromosomes in the genome.
 
+## One step denoising + peak calling command
 ```
-python $atacworks/scripts/peaksummary.py \
-    --peakbw output_latest/NK_inferred.peaks.bw \
-    --trackbw output_latest/NK_inferred.track.bw \
-    --prefix output_latest/NK_inferred.peak_calls \
-    --minlen 20
+bash Atacworks/scripts/run_inference.sh -bw <path to bigWig file with test ATAC-seq data> -m <path to model file> -f <path to chromosome sizes file> -o <output directory> -c <path to folder containing config files (optional)>
```

Comment:
Are we going to replace this with the new Atacworks denoise command?

Reply:
Yes, in the PR that does all of it in one step, I will replace this command.